src/common/reducers/entities.js

```diff
@@ -0,0 +1,62 @@
+import merge from 'lodash/merge';
+
+import * as ActionTypes from '../actions/repositories';
+
+// export only for testing
+export function repository(state={
+  __isSelected: false,
```

Comment:
Sounds good. But, if API is `snake_case`, so should be `is_selected`. Unless normalize can do camelCasing too for whitelisted props...